pack/Makefile

```diff
@@ -14,6 +14,8 @@ else ifneq (,$(wildcard /etc/debian_version))
 include $(PACKDIR)/deb.mk
 else ifneq (,$(wildcard /etc/alpine-release))
 include $(PACKDIR)/apk.mk
+else ifneq (,$(wildcard /etc/os-release))
+include $(PACKDIR)/rpm.mk
```

Comment:
This file is not specific for SuSE. It exists on my Gentoo host, for example. Can you look for other ways?  Preferably something like `ifneq (,$(wildcard /etc/suse-release))`.

Reply:
Right, I've changed it to check:
```
else ifneq (,$(shell grep "^ID=\"opensuse-leap\"" /etc/os-release))
```